src/main/kotlin/com/jetbrains/micropython/nova/ConnectAction.kt

```diff
@@ -63,7 +63,7 @@ suspend fun doConnect(fileSystemWidget: FileSystemWidget) {
             val result = Messages.showIdeaMessageDialog(
                 fileSystemWidget.project,
                 msg,
-                "Cannot Connect",
+                "Can Not Connect",
```

Comment:
_Can Not_ is not suitable here
https://www.grammarly.com/blog/commonly-confused-words/cannot-or-can-not/

Reply:
> _Can Not_ is not suitable here https://www.grammarly.com/blog/commonly-confused-words/cannot-or-can-not/

I did not know about this specific English grammar rule regarding cannot and can not, but it makes sense now that I know it. I will submit a new commit.